data/data/bootstrap/files/etc/systemd/system/node-image-pivot.target

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Node Image Pivot Target
```

Comment:
Should we replace the term Pivot? We talked a bit about this but maybe this would be an opportunity to use the bootc terminology?

```suggestion
Description=Switch Node Image Target
```

Reply:
Thinking of using the "overlay" terminology instead (e.g. `node-image-overlay.service`, "Node Image Overlay Target") since it's more accurate and distinct from switch/pivot which traditionally implied a reboot.